### ISSUE TITLE
How to unset options in Interproscan?

### ISSUE THREAD

**Issue by gbdias** [NBISweden/pipelines-nextflow#102]
The interproscan module currently runs the tool with many optional arguments activated.

https://github.com/NBISweden/pipelines-nextflow/blob/8b6c7faf72997ba865ac2992f4abfee12a9ec37a/config/functional_annotation_modules.config#L18C4-L27C6

How do I unset options like: `--goterms` and `-pa`?

**Comment by gbdias** [comment]
Ok great. So if I want all current args except one, I still need to provide the full list?

**Comment by mahesh-panchal** [comment]
No, unfortunately Nextflow doesn't have inheritance, so assigning to a variable overrides it in the config. If you want all except one, then yes, you provide that full list leaving out the one you don't need.
